jest.config.ts

```diff
@@ -1,9 +1,12 @@
-const { getJestProjects } = require('@nx/jest');
+import { getJestProjects } from '@nx/jest';
+import type { Config } from 'jest';
 
-export default {
+const config: Config = {
   projects: getJestProjects(),
   collectCoverage: true,
   coverageDirectory: '<rootDir>/coverage',
   coverageReporters: ['html', 'text', 'lcov'],
   reporters: ['./jest.reporter.js'],
 };
+
+export default config;
```

Comment:
Refactored jest config and preset to use typed values. Initially this was for esm support but it was removed, so i just kept the types